cpmpy/transformations/linearize.py

```diff
@@ -423,6 +409,10 @@ def only_positive_bv(lst_of_expr):
                         new_arg, cons = get_or_make_var(1 - arg)
```

Comment:
I know it is not part of this PR currently, but it should be fixed; and this seems like a good place to do it. This should be `1 - arg._bv`